spk/nzbhydra/Makefile

```diff
@@ -0,0 +1,48 @@
+SPK_NAME = nzbhydra
+SPK_VERS = 0.0.1a78
+SPK_REV = 1
+SPK_ICON = src/nzbhydra.png
+DSM_UI_DIR = app
+
+DEPENDS  = 
+SPK_DEPENDS = "python>2.7.9:git"
+
+MAINTAINER = SynoCommunity
```

Comment:
You can put your own name here as you're the original developer of the package. I assume you'll continue to maintain the package.